### PR TITLE
replace BashColor singleton with colorize function

### PR DESCRIPTION
**Pull request by JoshuaRLi**
AFAIK the BashColor singleton exists to toggle color enable/disable for easier testing, but it can indeed be replaced with a more lightweight and reusable `colorize` function.

Ideally, tests should be written to compare colors so the UX can be consistent - I won't be including that in this branch, and instead opted to just strip the ANSI escape sequences away for now since that was essentially what was being done before.

### CODE REVIEW

File: testing/util.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import re
+
+# https://stackoverflow.com/questions/14693701/how-can-i-remove-the-ansi-escape-sequences-from-a-string-in-python
```
Comment (KevinHock) [inline testing/util.py]: Wow, Martijn Pieters strikes again.

Comment (JoshuaRLi) [reply testing/util.py]: lol

File: tests/main_test.py [hunk 2]
```diff
@@ -253,7 +252,6 @@ def test_audit_same_file(self):
                 'No difference, because it\'s the same file!'
             )
 
-
```
Comment (KevinHock) [inline tests/main_test.py]: Still there, right?

**Comment by JoshuaRLi** [comment]
LOOOOL i added `test_audit_diff_not_enough_files` twice during the merge fuckery.

**Comment by JoshuaRLi** [comment]
Friday is a great day!

**Pull request merged**